src/platforms/node/common/profiling/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Profiling
+description: "Learn more about how to configure our Profiling integration and start profiling your code."
+---
+
+By default, Sentry error events will not get trace context unless you configure the scope with the transaction, as illustrated in the example below.
+
+<Note>
+
+Node.js profiling is currently in alpha. Alpha features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [profiling@sentry.io](mailto:profiling@sentry.io). If you’re adopting Profiling in a high-throughput environment, we recommend testing prior to deployment to ensure that your service’s performance characteristics maintain expectations.
+
+</Note>
+
+```javascript
+const Sentry = require("@sentry/node");
+// Note: You MUST import @sentry/tracing package before @sentry/profiling-node
+require("@sentry/tracing");
+const {ProfilingIntegration} = require("@sentry/profiling-node");
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [
+    // Add our Profilling integration
+    new ProfilingIntegration(),
+  ],
+  tracesSampleRate: 1.0,
+  // Set sampling rate for profiling
+  profilesSampleRate: 1.0,
+});
+
+const transaction = Sentry.startTransaction({
+  op: "transaction",
+  name: "My Transaction",
+});
+
+// Any code executed between startTransaction and transaction.finish
+// will now be automatically profiled.
+
+transaction.finish();
+```
+
+
+## How does it work?
+
+Under the hood, sentry profiler leverages V8's [CpuProfiler](https://v8docs.nodesource.com/node-18.2/d2/d34/classv8_1_1_cpu_profiler.html) to collect stack samples. This means that sentry/profiling-node is written as a [native addon](https://nodejs.org/docs/latest-v18.x/api/addons.html) for Node and will not run in environments like Deno or Bun. Profiling enhances tracing by providing profiles for the individual transactions - this allows you to look at higher level performance information like transaction and span durations before diving deeper and looking at profiles.
+
+## Runtime flags
+
+Profiling package enables you to run the profiler in either eager or lazy [mode](https://v8docs.nodesource.com/node-18.2/d2/dc3/namespacev8.html#a874b4921ddee43bef58d8538e3149374). During our initial tests, we have seen that the default profiling mode (lazy) can cause calls to startProfiling to take a long time - sometimes in the range of couple hundred milliseconds. We found that this occurs when we are starting a profile when no other profiles are started. This is the benefit of lazy logging, it adds no overhead when no profiles are running.
+
```

Comment:
```suggestion
## Runtime Flags

The Profiling package enables you to run the profiler in either [eager or lazy mode](https://v8docs.nodesource.com/node-18.2/d2/dc3/namespacev8.html#a874b4921ddee43bef58d8538e3149374). During initial tests, when we started a profile while no other profiles were running, we saw that the default profiling mode (lazy) sometimes caused startProfiling calls to take a long time, (sometimes in the range of a couple hundred milliseconds). The benefit of lazy logging is that it doesn't add any overhead when profiles aren't running.

```
Hoping that my edits didn't change the meaning of what you were trying to say :)